app/models/document_set_statistic.rb

```diff
@@ -69,23 +69,10 @@ def last_days_clause(last_days, column = "created_at")
     return clause
   end
   def get_stats_hash(start_date=nil, end_date=nil)
-    deeds = {
-      Deed::ARTICLE_EDIT => 0,
-      Deed::PAGE_TRANSCRIPTION => 0,
-      Deed::PAGE_EDIT => 0,
-      Deed::PAGE_INDEXED => 0,
-      Deed::NOTE_ADDED => 0,
-      Deed::PAGE_TRANSLATED => 0,
-      Deed::PAGE_TRANSLATION_EDIT => 0,
-      Deed::OCR_CORRECTED => 0,
-      Deed::NEEDS_REVIEW => 0,
-      Deed::TRANSLATION_REVIEW => 0,
-      Deed::TRANSLATION_INDEXED => 0,
-      Deed::WORK_ADDED => 0,
-    }
+    deeds = DeedType.generate_zero_counts_hash
```

Comment:
Having methods available on the `DeedType` class lets us access all DeedTypes where and when we need them. It was pretty handy here for replacing this hard-coded hash with a shiny new method.